test/cluster/access_control_test.go

```diff
@@ -1600,6 +1601,199 @@ var _ = Describe(
 							},
 						)
 
+						It(
+							"SecurityDisable: should disable security in running cluster",
+							func() {
+								accessControl := &asdbv1.AerospikeAccessControlSpec{
+									Roles: []asdbv1.AerospikeRoleSpec{
+										{
+											Name: "profiler",
+											Privileges: []string{
+												"read-write-udf.test.users",
+												"write",
+											},
+											Whitelist: []string{
+												"8.8.0.0/16",
+											},
+										},
+									},
+									Users: []asdbv1.AerospikeUserSpec{
+										{
+											Name:       "admin",
+											SecretName: test.AuthSecretNameForUpdate,
+											Roles: []string{
+												"sys-admin",
+												"user-admin",
+											},
+										},
+
+										{
+											Name:       "profileUser",
+											SecretName: test.AuthSecretNameForUpdate,
+											Roles: []string{
+												"data-admin",
+												"read-write-udf",
+												"write",
+											},
+										},
+									},
+								}
+
+								aerospikeConfigSpec, err := NewAerospikeConfSpec(latestImage)
+								if err != nil {
+									Fail(
+										fmt.Sprintf(
+											"Invalid Aerospike Config Spec: %v",
+											err,
+										),
+									)
+								}
+
+								aerospikeConfigSpec.setEnableSecurity(true)
+
+								aeroCluster := getAerospikeClusterSpecWithAccessControl(
+									clusterNamespacedName, accessControl,
+									aerospikeConfigSpec,
+								)
+								err = testAccessControlReconcile(
+									aeroCluster, ctx,
+								)
+								if err != nil {
+									Fail("Security should be enabled")
+								}
+
+								aerospikeConfigSpec, err = NewAerospikeConfSpec(latestImage)
```

Comment:
Can we not use the same `aerospikeConfigSpec` var above and manipulate security on it?
We are not changing anything apart from security. See at other places also if applicable